src/runge-kutta.jl

```diff
@@ -0,0 +1,315 @@
+# This file contains the implementation of explicit Runkge-Kutta
+# solver from (Hairer & Wanner 1992 p.134, p.165-169).
+
+include("tableaus.jl")
+
+# intermediate level interface
+
+"""
+
+A general Runge-Kutta stepper (it cen represent either, a fixed step
+or an adaptive step algorithm).
+
+"""
+immutable RKStepper{Kind,tab_name,T<:Number} <: AbstractStepper
+    tableau::Tableau
+    function RKStepper()
+        tab = eval(tab_name)
```

Comment:
I haven't been active here for a long time, so I might have missed something here, but how about having a (const) dictionary holding the tableaus, instead of defining them as constants? That way you could do `tab = tableaus[tab_name]` instead.